app/user/rest.py

```diff
@@ -133,6 +130,21 @@ def update_user_attribute(user_id):
     return jsonify(data=user_to_update.serialize()), 200
 
 
+def set_reply_to(recipient, template):
+    # if is_uk_phone_number(recipient):
```

Comment:
Is all this commented out code supposed to be here?

Reply:
remove it